pom.xml

```diff
@@ -36,7 +36,7 @@
     <properties>
         <gravitee-bom.version>1.4</gravitee-bom.version>
 
-        <gravitee-gateway-api.version>1.29.1</gravitee-gateway-api.version>
+        <gravitee-gateway-api.version>1.31.0-SNAPSHOT</gravitee-gateway-api.version>
```

Comment:
When merging on `master` or support branches dependencies must always have a stable version to ensure consistency of the builds and so of the generated artifact